### ISSUE TITLE
[Green][NETSDKE2E] [Loc] some list names are not localized in output window list.

### ISSUE THREAD

**Issue by yinjiehe01** [microsoft/vscode-dotnettools#1551]
### Describe the Issue

Affect language: TK JP SC DE
some list names are not localized.
![Image](https://github.com/user-attachments/assets/2059ae4b-be14-45d5-8293-a3e73da5ae0d)
![Image](https://github.com/user-attachments/assets/e2297515-735d-4f94-9b72-3a37872873ed)

### Steps To Reproduce

1.Install VS Code from [Download Visual Studio Code - Mac, Linux, Windows](https://code.visualstudio.com/download#)
2.Install C# and CDK.
3.Create a project and open it in VS Code.
4.Open the output window and then open the drop-down box, obverse the list.

### Expected Behavior

All names should be localized.

### Environment Information

C# 2.53.12: [Pipelines - Run 2.53.12](https://dev.azure.com/dnceng/internal/_build/results?buildId=2562558&view=results)
C# Devkit 1.12.25: [Pipelines - Run 1.12.25+07decd6e14](https://dev.azure.com/devdiv/DevDiv/_build/results?buildId=10410244&view=results)
OS: Linux
VS Code:1.42.2
.NET install tool:2.2.0
Affect language: TK JP SC DE

**Comment by arunchndr** [comment]
@Cosifne can we fix the C# side and hand it to @nagilson to handle the .NET install tool?

**Comment by nagilson** [comment]
Yes, for the top string we own that and it would have to be localized in our extension. cc @marcpopMSFT

**Comment by Cosifne** [comment]
@nagilson all strings here are localized on our side. You might want to localize the '.NET install tool' here.

**Comment by nagilson** [comment]
Thank you for the alert. All of our strings are external and need a completely new implementation of localization. It will take a while, there are hundreds or thousands of strings to localize. But if the SDK and other parts are localized, we can prioritize that.
